apps/gbridge/widget.js

```diff
@@ -1,4 +1,6 @@
 (() => {
+  const storage = require('Storage');
+  let settings;
```

Comment:
Any chance of just storing the one setting you're interested in? By storing all settings here you're wasting quite a lot of memory for all apps, just to read the value of one boolean.